packages/api/endpoints/elasticsearch.js

```diff
@@ -236,11 +236,11 @@ async function indexFromDatabase(req, res) {
       },
       esHost: process.env.ES_HOST,
       esRequestConcurrency: process.env.ES_CONCURRENCY,
-      stackName,
-      systemBucket,
-      dynamoTableName: tableName,
-      knexConfig,
     },
+    stackName,
```

Comment:
Do we have/can we add some sort of test to make sure that we don't break this again? It's hard because the endpoint just triggers an async operation, which is probably mocked in the endpoint test, so we'd need an integration test to prove that an index from database operation can be started?

Reply:
Yeah I can do that.

Reply:
If you've got ideas on how to clean this up after it's started. I'm all ears.  I don't see any stop task or delete index operations I have access too.